deploy/docs/Installation_with_Helm.md

```diff
@@ -306,3 +306,22 @@ kubectl delete secret sumologic
 ```
 
 and the associated hosted collector can be deleted in the Sumo Logic UI.
+
+### Removing the kubelet Service
+
+The Helm chart uses the Prometheus Operator to manage Prometheus instances.
+This operator creates a Service for scraping metrics exposed by the kubelet, which isn't removed by the chart uninstall process.
```

Comment:
Mention here the `values.yaml` config name https://github.com/prometheus-community/helm-charts/blob/kube-prometheus-stack-12.10.0/charts/kube-prometheus-stack/values.yaml#L1344-L1349 i.e. `prometheusOperator.kubeletService`

EDIT: hmm that's being done already below 🤔 maybe mention that here as well?